lib/puppet/functions/dnsquery/a.rb

```diff
@@ -3,7 +3,7 @@
 #
 # An optional lambda can be given to return a default value in case the
 # lookup fails. The lambda will only be called if the lookup failed.
-Puppet::Functions.create_function(:dns_a) do
```

Comment:
We should first create a version where theses old functions emit a deprecation warning